### PR TITLE
Add forms

### PR DESCRIPTION
**Pull request by LarryIVC**
## ***In this pull request, I did the following:***

- [ ] Added a branch.
- [ ] Created a method current_user.
- [ ] Created forms to add a post.
- [ ] Created a form to add a comment to the post.
- [ ] Created a button to add a like to a post.
- [ ] Check the linter.

### CODE REVIEW

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
+    </figure>
+    <%= link_to user_path(user.id) do  %>
+      <section class='user-info'> 
+        <h2>User: <%= user.name %></h2>
+        <p>Number of post:<strong> <%= user.post_counter || 0 %> </strong> </p>
+      </section>
+    <% end %>
 </article>
 <% end %>
+<%= link_to "Add a post", new_user_post_path(@current_user), class: 'button w120' %>
```
Comment (zieeco) [inline app/views/users/index.html.erb]: - Hey, here's this error thrown from this line. Please, kindly try to fix it. After this, your app should still meet the project requirements.


image

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @LarryIVC ,




Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️

## Required Changes ♻️
_Check the comments under the review._




## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._**


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/users/index.html.erb (hunk 1 above)
Comment (LarryIVC) [reply app/views/users/index.html.erb]: Hi @zieeco, this is because there are no users and the method returns a null.

**Review by OyePriscilla — APPROVED** [review]
Hi @LarryIVC 👏 👏, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Added post functionality worked well ✔️ 
Created a method called current_user in your ApplicationController ✔️ 
Created forms to perform the given functions ✔️ 
First `User` can add likes to Posts ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by LarryIVC** [comment]
Thanks, @OyePriscilla.

**Pull request merged**